nionswift_plugin/nion_experimental_4dtools/ComputationUI.py

```diff
@@ -59,15 +72,15 @@ def __display_item_changed(self, display_item):
                 for result in computation.results:
                     if result.specifier.get('uuid') == str(data_item.uuid):
                         computations_involved.append(computation)
-                if computation.source:
-                    if str(computation.source.uuid) == str(data_item.uuid):
-                        computations_involved.append(computation)
+                src = computation.get_input('src')
+                if src and str(src.uuid) == str(data_item.uuid):
+                    computations_involved.append(computation)
```

Comment:
The 'src' and `.source` are not the same thing. One refers to another object in Swift that "owns" the computation. The other refers to the variable named "src" in the computation inputs. Not all computations will have "src" as an input. Is this change intentional?

Reply:
I wasn't aware of the exact difference between the two and I don't think it is beneficial to use 'src' over '.source' so I changed it back.